src/Test/Runner.elm

```diff
@@ -374,9 +373,9 @@ type Shrinkable a
 {-| Given a fuzzer, return a random generator to produce a value and a
 Shrinkable. The value is what a fuzz test would have received as input.
 -}
-fuzz : Fuzzer a -> Random.Pcg.Generator ( a, Shrinkable a )
```

Comment:
This is a major change, and the only one in this PR. Since this is an advanced function I think we should have it `Debug.crash`, and long term move to a `Result`. Also, `ValidFuzzer` is an implementation detail and shouldn't leak to public API. (If it *is* hidden properly, this function won't be callable!)